ci/do_ci.sh

```diff
@@ -32,7 +32,10 @@ fi
 
 function collect_build_profile() {
   declare -g build_profile_count=${build_profile_count:-1}
-  mv -f "$(bazel info "${BAZEL_BUILD_OPTIONS[@]}" output_base)/command.profile.gz" "${ENVOY_BUILD_PROFILE}/${build_profile_count}-$1.profile.gz" || true
+  mv -f \
+     "$(bazel info "${BAZEL_BUILD_OPTIONS[@]}" output_base)/command.profile.gz" \
+     "${ENVOY_BUILD_PROFILE}/${build_profile_count}-$1.profile.gz" \
+      || true
```

Comment:
```suggestion
      || :
```

Reply:
will true more meaningful or easy to read? I didn't know much about the bash, so not sure.

Reply:
this is the more correct way, and as i corrected a lot of these previously (hopefully) more consistent

`:` means do nothing in bash - `true` means something so is less correct here